UofT-Frosh-Orientation/orientation-website
# 661 register now button

## **Contribution:**


### **Issue:**

Closes #661 

### **Accomplishments:**

- "Register Now" button is added to the home page
- button is visible without scrolling
- after logging in -> make sure the "Register Now" button changes to "View Profile"

### **Visuals:** 

Any related visuals.

## **Details:**

### **Expected Behavior:**

- see accomplishments

### **Testing Steps:**

Steps required to get expected behavior.

- run frontend with docker
- if logged in -> View Profile
- if not logged in -> Register Now

### **Complications:**

### **Resources:**

Any additional resources used outside this issue.
